feature/gnoi/system/tests/copying_debug_files_test/copying_debug_files_test.go

```diff
@@ -18,20 +18,66 @@ import (
 	"testing"
 	"time"
 
+	"github.com/openconfig/featureprofiles/internal/attrs"
 	"github.com/openconfig/featureprofiles/internal/fptest"
 	hpb "github.com/openconfig/gnoi/healthz"
-	spb "github.com/openconfig/gnoi/system"
+	gnps "github.com/openconfig/gnoi/system"
 	tpb "github.com/openconfig/gnoi/types"
 	"github.com/openconfig/ondatra"
+	"github.com/openconfig/ondatra/gnmi"
+)
+
+var processName = map[ondatra.Vendor]string{
+	ondatra.ARISTA:  "bgp",
+	ondatra.CISCO:   "bgp",
+	ondatra.JUNIPER: "rpd",
+	ondatra.NOKIA:   "bgp",
+}
+var component = map[ondatra.Vendor]string{
+	ondatra.ARISTA:  "Chassis",
+	ondatra.CISCO:   "Chassis",
+	ondatra.JUNIPER: "CHASSIS0",
+	ondatra.NOKIA:   "Chassis",
+}
+
+const (
+	ipv4PrefixLen = 30
 )
 
 var (
-	bgpProcName = map[ondatra.Vendor]string{
-		ondatra.NOKIA:  "sr_bgp_mgr",
-		ondatra.ARISTA: "bgp",
+	dutPort1 = attrs.Attributes{
+		Desc:    "dutPort1",
+		IPv4:    "192.0.2.1",
+		IPv4Len: ipv4PrefixLen,
+	}
+
+	atePort1 = attrs.Attributes{
+		Name:    "atePort1",
+		IPv4:    "192.0.2.2",
+		IPv4Len: ipv4PrefixLen,
+	}
+
+	dutPort2 = attrs.Attributes{
+		Desc:    "dutPort2",
+		IPv4:    "192.0.2.5",
+		IPv4Len: ipv4PrefixLen,
+	}
+
+	atePort2 = attrs.Attributes{
+		Name:    "atePort2",
+		IPv4:    "192.0.2.6",
+		IPv4Len: ipv4PrefixLen,
 	}
 )
```

Comment:
Why do we need these IP definitions?

Reply:
Not required. Removed from latest commit